config/default.yml

```diff
@@ -18,6 +18,7 @@ Flexport/GlobalModelAccessFromEngine:
   GlobalModelsPath: app/models/
   DisabledEngines: []
   AllowedGlobalModels: []
+  AdditionalGlobalModels: []
```

Comment:
The relationship between this param and the one above is not clear. Can we instead accomplish the same goal by making `GlobalModelsPath` take either a single string (as it does now) or a list (to support the case where models are stored in other directories)?